src/inc/apithreadstress.cpp

```diff
@@ -67,10 +67,11 @@ APIThreadStress::~APIThreadStress()
     if (m_threadCount > 0)
     {
         HANDLE *p = m_hThreadArray;
-        HANDLE *pEnd = p + m_threadCount;
+        HANDLE *pEnd;
```

Comment:
Could you please move the whole line inside `if (p != NULL)` ?

Reply:
done